analyzers/tests/SonarAnalyzer.UnitTest/Common/RuleLoaderTest.cs

```diff
@@ -33,12 +33,25 @@ public class RuleLoaderTest
         public void GivenNonXmlFile_RuleLoader_Throws()
         {
             var sut = new RuleLoader();
-            sut.Invoking(x => x.GetEnabledRules(@"Common/Resources/input.txt")).Should().Throw<XmlException>();
+            sut.Invoking(x => x.GetEnabledRules("not xml")).Should().Throw<XmlException>();
         }
 
         [TestMethod]
-        public void GivenAnExistingSonarLintFile_RuleLoader_LoadsActiveRules() =>
-            CollectionAssert.AreEqual(new RuleLoader().GetEnabledRules(@"Common/Resources/SonarLint.xml").ToArray(),
-                                      new[] { "S1067" });
+        public void GivenSonarLintXml_RuleLoader_LoadsActiveRules()
+        {
+            const string content = @"<?xml version=""1.0"" encoding=""UTF-8""?>
```

Comment:
This used to be in the deleted SonarLint.xml file.
Since this class no longer reads anything from disk, I thought it would make the test clearer if the text was in plain string.